audio/nrsc5/Portfile

```diff
@@ -32,11 +32,16 @@ depends_build-append \
                     path:bin/cmake:cmake \
```

Comment:
Remove `path:bin/cmake:cmake` here; the cmake portgroup has added this for you already.